src/remote/manager.go

```diff
@@ -140,10 +140,14 @@ func (u *UnitOfWork) createOrUpdate(objectState model.ObjectState, changedFields
 	switch v := object.(type) {
 	case *model.Config:
 		v.ChangeDescription = u.changeDescription
-		changedFields = append(changedFields, "changeDescription")
+		if len(changedFields) > 0 {
+			changedFields = append(changedFields, "changeDescription")
```

Comment:
Tu bola chyba, .. ak je `changedFields` prazdne, tak to znamena, ze sa ulozia vsetky polozky, a teda nie je potrebne nic pridavat.